pylib/cqlshlib/copyutil.py

```diff
@@ -2101,6 +2101,13 @@ def convert_map(val, ct=cql_type):
             return ImmutableDict(frozenset((convert_mandatory(ct.subtypes[0], v[0]), convert(ct.subtypes[1], v[1]))
                                  for v in [split(split_format_str % vv, sep=sep) for vv in split(val)]))
 
+        def convert_vector(val, ct=cql_type):
+            string_coordinates = val.strip("[]").split(',')
+            if len(string_coordinates) != ct.vector_size:
+                raise ParseError("The length of given vector value '%d' is not equal to the vector size from the type definition '%d'" % (len(string_coordinates), ct.vector_size))
+            float_vector = [float(v) for v in string_coordinates]
```

Comment:
Semi-nit: you can just return the built vector here rather than assigning it to "float_vector" and returning that.  I don't raise it just to be pedantic; such an impl would be more consistent with the rest of the functions in copyutil.py.

Reply:
That's absolutely right, considering the current shape of that method there is no value in keeping this intermediary variable.